cspell.json

```diff
@@ -81,6 +81,7 @@
         "toctree",
         "topness",
         "venv",
+        "visualise",
```

Comment:
Sorry my bad, should be "visualize" in the `vizualisation.ipynb`, because [we use American English](https://pwa.readthedocs.io/projects/expertsystem/en/0.3.0-alpha/contribute.html#spelling). So please remove this line and change to "visualize" in the notebook.